GOVERNANCE.md

```diff
@@ -153,7 +151,14 @@ Once a PR is ready to be landed, the CPC member who lands the pull request shoul
 * Add the member to the `cpc-private` email list and directory by opening a PR against the [OpenJS Foundation directory](https://github.com/openjs-foundation/directory-private/blob/master/groups/cross-project-council.yml)
 * Introduce the new member at the next CPC meeting.
 
-Former Voting members whose terms have just ended will automatically become Regular members, unless they indicate otherwise.
+_Note: Former Voting members whose terms have just ended will automatically become Regular members, unless they indicate otherwise._
+
+## Definition of an Active OpenJS Collaborator
+
+### To be considered An active OpenJS Collaborator, one of the following is required:
+
+- An active member of a project community or collaboration space which is defined as having recent, sustained contributions to the project during the past 90 days.
```

Comment:
Unless "contributions" are defined elsewhere in this doc, I think we'd want to define them here in order to clarify that they aren't limited to code contributions.

Reply:
I've clarified that a little bit further @tobie. Let me know if you think this is satisfactory. Thanks.

Reply:
Added a few more examples.